elastalert/elastalert.py

```diff
@@ -1314,6 +1314,8 @@ def handle_rule_execution(self, rule):
             self.handle_error("Error running rule %s: %s" % (rule['name'], e), {'rule': rule['name']})
         except Exception as e:
             self.handle_uncaught_exception(e, rule)
+            """ After the exception has been handled, exit without raising an exception """
+            os._exit(0)
```

Comment:
Should we rather exit with exitcode 1 to differentiate it from normal exit when the host goes down etc?

Reply:
Didn't look at it from this perspective, but I can't not to agree. 

https://github.com/auto1-oss/elastalert/pull/22/files#diff-de8c5a47817fa23d1afa305e6d3c3e8ee717e7236c4278a57df4ab30d4ecddcaR1318